src/plugins/vis_type_timeline/server/lib/config_manager.ts

```diff
@@ -36,11 +36,16 @@ import { configSchema } from '../../config';
 
 export class ConfigManager {
   private opensearchShardTimeout: number = 0;
-  private graphiteUrls: string[] = [];
+  private graphiteAllowedUrls: string[] = [];
+  private graphiteBlockedIPs: string[] = [];
 
   constructor(config: PluginInitializerContext['config']) {
     config.create<TypeOf<typeof configSchema>>().subscribe((configUpdate) => {
-      this.graphiteUrls = configUpdate.graphiteUrls || [];
+      this.graphiteAllowedUrls = configUpdate.graphiteAllowedUrls || [];
+    });
+
+    config.create<TypeOf<typeof configSchema>>().subscribe((configUpdate) => {
```

Comment:
NIT: i think this is created another subscription to listen to updates, probably can just put line 48 under line 44 right

Reply:
Nice. Will do.